src/ controllers/trainee/Controller.ts

```diff
@@ -0,0 +1,62 @@
+import { Request, Response } from 'express';
+
+class TraineeController {
+    static instance;
+    static getInstance = () => {
+        if (TraineeController.instance) {
+            return TraineeController.instance;
+        }
+        TraineeController.instance = new TraineeController();
+        return TraineeController.instance;
+    }
+create = (req: Request, res: Response ) => {
+    console.log(':::::::Create Trainee:::::::');
+    res.send({
+        status: 'Ok',
+        message: 'Trainee added',
+        details: [{
+        id: 1,
+        name: 'Ajay',
+        address: 'Rohtak',
+}]
+});
+}
+list = (req: Request, res: Response ) => {
+    console.log(':::::::list Trainee:::::::');
+    res.send({
+        status: 'Ok',
+        message: 'Trainee added',
+        details: [{
+        id: 2,
+        name: 'Rahul',
+        address: 'Shahdra',
+}]
+});
+}
+Update = (req: Request, res: Response ) => {
+    console.log(':::::::Update Trainee:::::::');
```

Comment:
1. do proper validation if not id then throw an error with 400 code and error message